kernel/kernel-config-server/README.md

```diff
@@ -148,7 +161,7 @@ spring.cloud.config.uri=http://<config-host-url>:<config-port>
 spring.cloud.config.label=<git-branch>
 spring.application.name=<application-name>
 spring.cloud.config.name=<property-file-to-pick-up-configuration-from>
-spring.profiles.active=<active-profile>
+spring.profiles.active=composite
```

Comment:
this could be native or composite based on where the team in running the config server

Reply:
we can support native properties through composite profile as below
```
export SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_TYPE=native
export SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_URI=<path for file>
```